go/sqldb/sql_error.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sqldb
+
+import "fmt"
+
+// SqlError is the error structure returned from calling a db library function
+type SqlError struct {
```

Comment:
I feel this level of redundancy might be ok. sqldb package is open to have other error types and SqlError is just one of them. Although sqldb only has one error: SqlError for now.